src/http/api/views/account.follows.view.ts

```diff
@@ -141,10 +183,15 @@ export class AccountFollowsView {
             return error('not-an-actor');
         }
 
+        const followeeAccount = await this.db('accounts')
+            .where('ap_id', actor.id?.toString() || '')
+            .first();
+
         let page: CollectionPage | null = null;
 
         try {
-            if (next !== '') {
+            //next would be a number only in case of non-paginated follow lists
```

Comment:
I'm a little bit confused by this, but I think instead of passing around a `string` which is actually a `URL` or a `number` - which should just deal with those types directly.

So we could have `next: URL | number | null`